app/src/main/java/com/kickstarter/ui/SharedPreferenceKey.java

```diff
@@ -5,6 +5,7 @@ private SharedPreferenceKey() {}
 
   public static final String ACCESS_TOKEN = "access_token";
   public static final String CONFIG = "config";
+  public static final String FIRST_SESSION = "first_session";
```

Comment:
Do you need to persist this outside of Qualtrics? I figured since you're able to get the value stored in Qualtrics' properties you could check if there is already something there otherwise set it (for the first time user thing).

Reply:
I learned the hard way that Qualtrics calls the initialization block more than once 😒 so that's why I'm moved it here